Rozdział 8/Hangman2.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Hangman2!</title>
+</head>
+<body>
+    <script>
+    
+    var words = ["javascript", "monkey", "amazing", "pancake", "tractor", "blablabla"];
+
+    
+    var pickWord = function() {
+        return words[Math.floor(Math.random() * words.length)];
+    };
+
+    var setupAnswerArray = function (word) {
+        var answerArray = [];
+    for (var i = 0; i < word.length; i++) {
```

Comment:
wcięcia! `for` jest na tej samej wysokości u Ciebie co `var setupAnswerArray = function (word) {`, a przecież znajduje się wewnątrz tej funkcji. To jest niby pierdoła, ale bardzo zakłóca czytanie i zrozumienie kodu.